packages/plugin-ext/src/main/browser/comments/comment-thread-widget.tsx

```diff
@@ -231,7 +235,8 @@ export class CommentThreadWidget extends BaseWidget {
             if (this._commentThread.comments && this._commentThread.comments.length) {
                 const onlyUnique = (value: Comment, index: number, self: Comment[]) => self.indexOf(value) === index;
                 const participantsList = this._commentThread.comments.filter(onlyUnique).map(comment => `@${comment.userName}`).join(', ');
-                label = `Participants: ${participantsList}`;
+                const resolutionState = (this._commentThread.state === CommentThreadState.Resolved) ? '(Resolved)' : '(Unresolved)';
```

Comment:
minor: the parentheses are unnecessary

```suggestion
                const resolutionState = this._commentThread.state === CommentThreadState.Resolved ? '(Resolved)' : '(Unresolved)';
```